static/app/components/contextData/index.tsx

```diff
@@ -154,13 +182,24 @@ function walk({
             preserveQuotes,
             withAnnotatedText,
             jsonConsts,
-            meta: meta?.[key]?.[''] ?? meta?.[key] ?? meta?.[''] ?? meta,
```

Comment:
If `value[key]` is an array, it is bad to use `meta: meta?.[key]?.['']`, since it means we can't access the meta of any items in the array